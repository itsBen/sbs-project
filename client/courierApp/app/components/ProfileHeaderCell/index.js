import React from 'react';
import { Image, StyleSheet, Text, View } from 'react-native';

import { defaultPaddings } from '../../config';

export default () => (
  <View style={styles.container}>
    <View style={styles.titleDescription}>
      <Text style={styles.titleText}>Anna</Text>
      <Text style={styles.descriptionText}>View and edit profile</Text>
    </View>
    <Image
      style={styles.profileImage}
      source={{ uri: 'https://facebook.github.io/react/img/logo_og.png' }}
    />
  </View>
);

const styles = StyleSheet.create({
  container: {
    flexDirection: 'row',
    paddingHorizontal: defaultPaddings.paddingHorizontal,
    paddingVertical: 20,
    alignItems: 'center',
    backgroundColor: '#fff',
  },
  titleDescription: {
    flex: 1,
  },
  titleText: {
    fontSize: 30,
    fontWeight: 'bold',
    marginBottom: 10,
  },
  descriptionText: {},
  profileImage: {
    width: 70,
    height: 70,
    borderRadius: 35,
  },
});
