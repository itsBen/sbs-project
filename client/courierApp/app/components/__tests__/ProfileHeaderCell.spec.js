/* eslint-env node, jest */
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import ProfileHeaderCell from '../ProfileHeaderCell';

it('renders correctly', () => {
  const tree = renderer.create(<ProfileHeaderCell />).toJSON();
  expect(tree).toMatchSnapshot();
});
