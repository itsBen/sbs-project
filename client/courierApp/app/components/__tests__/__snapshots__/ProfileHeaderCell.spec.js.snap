// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flexDirection": "row",
      "paddingHorizontal": 15,
      "paddingVertical": 20,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "fontSize": 30,
          "fontWeight": "bold",
          "marginBottom": 10,
        }
      }
    >
      Anna
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={Object {}}
    >
      View and edit profile
    </Text>
  </View>
  <Image
    source={
      Object {
        "uri": "https://facebook.github.io/react/img/logo_og.png",
      }
    }
    style={
      Object {
        "borderRadius": 35,
        "height": 70,
        "width": 70,
      }
    }
  />
</View>
`;
